### PR TITLE
feat: payment banners on landing and login pages

### PR DESCRIPTION
**Pull request by LinHuiqing**
## Problem

More marketing efforts for our payments feature!

Closes FRM-960, closes FRM-961

## Solution

**Breaking Changes** 

- [x] No - this PR is backwards compatible  

**Features**:

- Feature banner for payments and other future big features, used in the following places:
   - Login Page
   - Landing Page

**Improvements**:

- Added h5 and h6 textStyles
- Use constants for links to guide

**Bug Fixes**:

- Fixed colour of login page banner

## Before & After Screenshots

**BEFORE**:

Landing Page:

Screenshot 2023-06-06 at 5 32 19 PM

Login Page:

Screenshot 2023-06-06 at 5 32 28 PM

**AFTER**:

Landing Page:

Desktop:

Screenshot 2023-06-07 at 7 05 14 PM

Mobile:

Screenshot 2023-06-07 at 7 05 40 PM

Login Page:

Desktop:

Screenshot 2023-06-07 at 7 05 05 PM

Mobile:

Screenshot 2023-06-07 at 7 05 46 PM

## Tests

Check that new banners work:
- [ ] Landing page
   - [ ] Check that banner has the right copy
   - [ ] Check that the "Learn more" button opens the public FormSG payment guide in a new tab
- [ ] Login page
   - [ ] Check that banner has the right copy
   - [ ] Check that the "Learn more" button opens the public FormSG payment guide in a new tab

Check that the refactor that takes out the links as constants works with the existing payments copy things:
- [ ] Announcement Rollout Modal
   - [ ] If you've seen the announcement modal before, right click > Inspect > Applications. Delete local storage items prefixed with "has-seen-rollout-announcement-"
   - [ ] Check that the content in the announcement modal that pops up has the link to the payment directory slides
- [ ] What's New drawer
   - [ ] Check that the content in the What's New drawer has the link to the payment directory slides

### CODE REVIEW

**Comment by LinHuiqing** [comment]
@staceytan1998 seems like chromatic still isn't rendering it properly, could you check the copy on [staging-alt2.form.gov.sg](https://staging-alt2.form.gov.sg) instead?

**Comment by LinHuiqing** [comment]
for posterity, banners do not render properly on chromatic and have been (slack) approved by designer @staceytan1998 after checking the banners on [staging-alt2.form.gov.sg](https://staging-alt2.form.gov.sg).

File: frontend/src/components/Banner/Banner.tsx [hunk 1]
```diff
@@ -45,7 +47,7 @@ export const Banner = ({
 
   return (
     <Collapse style={{ overflow: 'visible' }} in={isOpen} animateOpacity>
-      <Box __css={styles.banner}>
+      <Box __css={styles.banner} bgColor={bannerColor}>
```
Comment (KenLSM) [inline frontend/src/components/Banner/Banner.tsx]: Might need to optionally add `bgColor` in the event that `__css={styles.banner}` has a default value. Not exactly sure how `__css` interacts with `bgColor: undefined`, but if under the hood, the styles are combined, `bgColor: undefined` might unset the value from `styles.banner`

File: frontend/src/features/rollout-announcement/Announcements.tsx [hunk 2]
```diff
@@ -8,7 +10,7 @@ export const NEW_FEATURES = [
     title: 'Collect payments on your form',
     description:
       'Citizens can now pay for fees and services directly on your form! We integrate with Stripe to provide reliable payments and hassle-free reconciliations. Payment methods we support include debit / credit cards and PayNow.',
-    learnMoreLink: 'https://go.gov.sg/formsg-payment-overview',
+    learnMoreLink: GUIDE_PAYMENTS_ENTRY,
```
Comment (KenLSM) [inline frontend/src/features/rollout-announcement/Announcements.tsx:13]: 👍

**Review by KenLSM — APPROVED** [review]
LGTM! With on comment on `bannerColor` props

File: frontend/src/components/Banner/Banner.tsx (hunk 1 above)
Comment (LinHuiqing) [reply frontend/src/components/Banner/Banner.tsx]: Thanks, done here [f14df7e](https://github.com/opengovsg/FormSG/pull/6431/commits/f14df7ed9f3f3dff84b57d81f7b36c67f75d330f)!

**Pull request merged**